index.html

```diff
@@ -28,52 +28,40 @@
     </header>
 
     <aside class="aside">
-        <nav>
-            <a href="#welcome-screen" class="btn aside__home-btn">
-                <img src="img/up-arrow.png" alt="Go up" class="home-btn-img" />
+        <nav class="btn aside__home-btn">
+            <a href="#welcome-screen" class="link">
+                ^
             </a>
         </nav>
     </aside>
     
     <main class="main">
-        <section id="welcome-screen" class="main--section main__welcome-screen" tabindex="0">
+        <section id="welcome-screen" class="main__section main__welcome-screen" tabindex="0">
```

Comment:
Generalna uwaga: zauważ, że element, który użyłeś nazwy się `section` i nazwałeś klasę `main__section`. 

Z jednej strony to niepotrzebna redundacja - może pisać selektory css odwołujący się do elementów [semantycznych](https://www.w3schools.com/html/html5_semantic_elements.asp) takich właśnie jak `section`. Tak samo masz np. `a` i klasa `link`
```
a { ... }
```

Po drugie - nie trzeba tak dosłownie brać BEM i restrykcyjnie, bo właśnie wpada się w pułapkę, że to co zakodowałeś jest mało reużywalne. Teoretycznie możesz użyć gdzieś indziej `main__section`, ale użycie tego po za `main`, będzie sprawiało wrażenie bałaganu czy nieczytelności.

Jeśli jednak chcesz restrykcyjnie trzymać się BEM, to warto pomyśleć nad nazwami klas, które bardziej coś mówią o tym elemencie, dzięki temu też unikniesz redudancji znaczeniowej. np. `landing-page-section` - co i tak jest średnią nazwą, ale, aby coś wymyślić musiałbym bardziej poznać Twoją stronę.